toolchain/sem_ir/formatter.cpp

```diff
@@ -855,6 +855,10 @@ class FormatterImpl {
     out_ << "}";
   }
 
+  auto FormatInstructionRHS(BindingPattern inst) -> void {
+    FormatArgs(inst.entity_name_id);
+  }
```

Comment:
Consider e.g. this line from `toolchain/check/testdata/tuple/two_elements.carbon`:

```
// CHECK:STDOUT: %.loc11_5: %.3 = binding_pattern v, @__global_init.%v
```

With the default formatter, it would instead be:

```
// CHECK:STDOUT: %.loc11_5: %.3 = binding_pattern @__global_init.%v
```

As you pointed out, conceptually a `binding_pattern` consists of a type and a name (in this case `v`), so it seems desirable for the pretty-printed form to include that name. If anything, arguably we should omit the `@__global_init.%v` since it's not really an operand of the inst (and as you pointed out, it's a forward reference to an inst that will come later).